src/explore-education-statistics-admin/src/pages/publication/components/LegacyReleasesTable.tsx

```diff
@@ -41,9 +44,15 @@ const getModalTitle = (confirmAction?: ConfirmAction): string => {
 };
 
 interface Props {
+  createRoute?: string; // EES-3217 remove when live
+  editRoute?: (id: string) => string; // EES-3217 remove when live
```

Comment:
Not a biggy, but would get into the convention of prefix things like this `TODO`. IDEs and VS Code extensions can detect these in the codebase more easily